src/CallMessage.ts

```diff
@@ -0,0 +1,123 @@
+/**
+ * Copyright © 2022 Twilio, Inc. All rights reserved. Licensed under the Twilio
+ * license.
+ *
+ * See LICENSE in the project root for license information.
+ */
+
+//@ts-ignore
+import { Constants } from './constants';
+import type { NativeCallMessageInfo } from './type/CallMessage';
+
+/**
+ * Provides access to information about a callMessage, including the call
+ * message content, contentType, messageType, and voiceEventSid
+ *
+ * @remarks
+ * Note that the callMessage information is fetched as soon as possible from the
+ * native layer, but there is no guarantee that all information is immediately
+ * available. Methods such as `CallMessage.getContent` or `CallMessage.getSid`
+ * may return `undefined`.
```

Comment:
In what scenario where these return undefined? How can a customer know when these are available?

Reply:
We have similar notes in `Call.ts` for getters. https://github.com/twilio/twilio-voice-react-native/blob/VBLOCKS-2464/feat-call-message/src/Call.tsx#L335

I will remove unless @mhuynh5757 objects?

Reply:
The concern is, if customers see this, they would likely have the same question.

Reply:
It's different for the `Call` objects since there are situations (i.e. outgoing calls) where sometimes values are `undefined` for lengths of time. However, since we construct the `CallMessage` objects ourselves for outgoing call messages, and we'll always have the values defined, we don't need `undefined` here.